src/MC/fix_rmc_partial.cpp

```diff
@@ -0,0 +1,1381 @@
+/* -*- c++ -*- ----------------------------------------------------------
+ * LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+ * https://www.lammps.org/, Sandia National Laboratories
+ * LAMMPS development team: developers@lammps.org
+ *
+ * Copyright (2003) Sandia Corporation.  Under the terms of Contract
+ * DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+ * certain rights in this software.  This software is distributed under
+ * the GNU General Public License.
+ *
+ * See the README file in the top-level LAMMPS directory.
+ * ------------------------------------------------------------------------- */
+
+/* ----------------------------------------------------------------------
+   Contributing author: Vishnu Raghuraman (UIUC)
+   Email: vishnura@illinois.edu
+------------------------------------------------------------------------- */
+
+#include "fix_rmc_partial.h"
+
+#include "angle.h"
+#include "atom.h"
+#include "bond.h"
+#include "comm.h"
+#include "compute.h"
+#include "dihedral.h"
+#include "domain.h"
+#include "error.h"
+#include "force.h"
+#include "improper.h"
+#include "json.h"
+#include "kspace.h"
+#include "modify.h"
+#include "neighbor.h"
+#include "pair.h"
+#include "text_file_reader.h"
+#include "update.h"
+
+using namespace LAMMPS_NS;
+using namespace FixConst;
+
```

Comment:
You may want to add here a "citeme" reminder to cite the most relevant publication describing the implementation of this fix.